### ISSUE TITLE
Problems with builder.py

### ISSUE THREAD

**Issue by martinholmer** [open-source-economics/policybrain-builder#82]
@hdoupe, Now that `builder.py` has been merged into the master branch, it needs to be fixed in several ways.

First, it doesn't comply with what used to be called the PEP8 coding style, which is automatically checked with every Tax-Calculator check-in on GitHub.   Here are the formatting problems:
```
policybrain-builder$ pycodestyle builder.py 
builder.py:36:1: E302 expected 2 blank lines, found 1
builder.py:51:80: E501 line too long (81 > 79 characters)
builder.py:74:80: E501 line too long (124 > 79 characters)
builder.py:75:80: E501 line too long (114 > 79 characters)
builder.py:81:80: E501 line too long (136 > 79 characters)
builder.py:105:5: E128 continuation line under-indented for visual indent
builder.py:106:5: E128 continuation line under-indented for visual indent
builder.py:107:5: E122 continuation line missing indentation or outdented
builder.py:108:5: E122 continuation line missing indentation or outdented
builder.py:109:5: E122 continuation line missing indentation or outdented
builder.py:110:5: E122 continuation line missing indentation or outdented
builder.py:111:5: E122 continuation line missing indentation or outdented
builder.py:114:5: E122 continuation line missing indentation or outdented
builder.py:117:5: E122 continuation line missing indentation or outdented
builder.py:119:1: E901 TokenError: EOF in multi-line statement
```
Just because we don't have GitHub configured to do these style checks in the policybrain-builder repo doesn't mean that we shouldn't strive in that repo to have programs that meet minimal coding-style standards.

Second, when I run the `builder.py` code through `pylint`, I get this error:
```
policybrain-builder$ pylint builder.py 
Using config file /Users/mrh/.pylintrc
************* Module builder
builder.py:74: [E0001(syntax-error), ] invalid syntax (<string>, line 74)
```
This is the same error as my Python 2.7 interpreter stopped on.  I know most of the OSPC developers don't think much of `pylint`, but in this case running the code through `pylint` found a syntax error that prevents the code from running.  That's a pretty easy way to check the code before actually trying to run the code in a full-blown test.

**Comment by hdoupe** [comment]
@martinholmer said:
> Just because we don't have GitHub configured to do these style checks in the policybrain-builder repo doesn't mean that we shouldn't strive in that repo to have programs that meet minimal coding-style standards.

I agree. I'll update the script to satisfy the code-style requirements.

> This is the same error as my Python 2.7 interpreter stopped on. I know most of the OSPC developers don't think much of pylint, but in this case running the code through pylint found a syntax error that prevents the code from running. That's a pretty easy way to check the code before actually trying to run the code in a full-blown test.

Ok, I used the new f-strings pretty liberally in this script. I think that's why `pylint` is barking at us. f-strings are only available in Python 3.6. When I wrote this, I didn't think about someone wanting to run it with Python 2. I'm happy to make the necessary changes to make it compatible with Python 2 if there is a demand for that.

**Comment by martinholmer** [comment]
@hdoupe said in policybrain-builder issue #82:
> Ok, I used the new f-strings pretty liberally in this script. I think that's why `pylint` is barking at us.
> f-strings are only available in Python 3.6. When I wrote this, I didn't think about someone wanting to run it with Python 2. I'm happy to make the necessary changes to make it compatible with Python 2 if there is a demand for that.

I know the OSPC projects plan to quite using Python 2.7 beginning in January 2019, but some of us haven't yet made the switch.  And more importantly, you've had you eye on flexibility with the idea that the `builder.py` script might be useful to other developers outside of the OSPC circle.  So, maybe they are still using Python 2.7 and would run into the same problem I ran into when trying to use the `builder.py` script.

**Comment by martinholmer** [comment]
@hdoupe said:
> Ok, I used the new f-strings pretty liberally in this script. I think that's why pylint is barking at us. f-strings are only available in Python 3.6. When I wrote this, I didn't think about someone wanting to run it with Python 2.

It's not just barking.  This is illegal code under Python 2.7, which is a show stopper for anybody interested in using your script who is still using Python 2.7.

**Comment by hdoupe** [comment]
@martinholmer said:
> I know the OSPC projects plan to quite using Python 2.7 beginning in January 2019, but some of us haven't yet made the switch. And more importantly, you've had you eye on flexibility with the idea that the builder.py script might be useful to other developers outside of the OSPC circle. So, maybe they are still using Python 2.7 and would run into the same problem I ran into when trying to use the builder.py script.

Ok, these are good points. I'll update PR #83 so that it is Python 2.7 compatible.

> It's not just barking. This is illegal code under Python 2.7, which is a show stopper for anybody interested in using your script who is still using Python 2.7.

Agreed, I'll make the updates.
